addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -385,6 +381,19 @@ body.editor_enable.editor_has_snippets {
                 > span {
                     flex: 0 0 auto;
                 }
+
+                &.we-full-width {
+                    flex-flow: row wrap;
+                    > input {
+                        flex: 0 0 100%;
+                    }
+                }
+                &:not(.we-full-width) {
+                    > we-title {
+                        min-width: 60%;
+                        margin-right: 5%;
+                    }
+                }
```

Comment:
- `o_we_full_width`
- Avoid the use of `:not`, you can add the margin 5% as a margin left on the input and remove it in full width since you already have a rule for the input there

Reply:
Alright, done.
I personally prefer
```scss
we-input {
    &:not(.we-full-width) {
        > we-title {
            min-width: 60%;
            margin-right: 5%;
        }
    }
}
```
then
```scss
we-input {
    > we-title {
        min-width: 60%;
        margin-right: 5%;
    }

    &.o_we_full_width {
        > we-title {
            min-width: unset;
            margin-right: 0%;
        }
    }
}
```
But I am fine with both